### ISSUE TITLE
[BUG] _CMCameraTarget Placement Issue

### ISSUE THREAD

**Issue by BJPickles** [insthync/UnityMultiplayerARPG_Cinemachine#1]
Currently _CMCameraTarget is placed at the root of the character. When the root of the character interacts with an uneven mesh, it moves the camera making it unplayable.

Video:
https://i.imgur.com/LOzBRRC.mp4

Need a way to define where _CMCameraTarget will hook onto, or control the Y of _CMCameraTarget.

**Comment by insthync** [comment]
For [this](https://github.com/insthync/UnityMultiplayerARPG_Cinemachine)?

You may just change codes for your project, it is open-source project.

**Comment by BJPickles** [comment]
Yeah, didn't want to add it to main MMORPGKIT issue because it wasn't directly related.
Integration works well but not sure how to change the _CMCameraTarget Y coordinate.

**Comment by insthync** [comment]
It is having only 2 script files, you should able to find it.

**Comment by BJPickles** [comment]
Okay, once I get it I can make a pull request.
It will just allow users to set the Y offset of  _CMCameraTarget, which means it won't clip with slight bumps in the ground.
